user_class/spec/user_spec.rb

```diff
@@ -42,4 +42,25 @@
       end
     end
   end
+
+  describe ".find" do
```

Comment:
```suggestion
  describe "#find" do
```

.find is ok if the definition is with self. S, basically . for class methods and # for instance methods.